proxy/proxy.go

```diff
@@ -213,6 +221,12 @@ func mapRequest(r *http.Request, rt *routing.Route, host string) (*http.Request,
 	u.Scheme = rt.Scheme
 	u.Host = rt.Host
 
+	backendURL, err := url.Parse(rt.Backend)
```

Comment:
why? applyFiltersToRequest is called, and it sets the Authorization header, then here in the maprequest, a new request object is created, and the cloneHeader copies the Authorization header among others. The end result is the same as if it was taken from the backend url.